### PR TITLE
Adds Hotel Urbano as an adopter

### PR DESCRIPTION
**Pull request by andrestc**
This PR adds Hotel Urbano as an adopter. We are currently using the project in our search api.

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ URLs if possible) and [submit a pull request][1]!
 * [Green Man Gaming](http://www.greenmangaming.com/)
 * [Gutefrage.net](http://www.gutefrage.net/)
   * ["Introducing Gutefrage.net's Service-Oriented Architecture"][2]
+* [Hotel Urbano](https://www.hotelurbano.com)
```
Comment (mosesn) [inline ADOPTERS.md]: could you add a trailing slash here for consistency?

**Comment by mosesn** [comment]
LGTM, except for a nit.  Glad to hear you're using it!

**Comment by andrestc** [comment]
Addressed your comment, @mosesn. There were others missing the trailing slash so i added on those too.

**Comment by mosesn** [comment]
Rad, thanks!

**Comment by vkostyukov** [comment]
This has been merged in [this commit](https://github.com/twitter/finagle/commit/ed082f5c673856c50797e3659aa23f07b92fe90d). Thanks @andrestc!

**Pull request closed without merging**